.gitmodules

```diff
@@ -1,3 +1,8 @@
 [submodule "OpenAssetIO"]
 	path = OpenAssetIO
 	url = https://github.com/OpenAssetIO/OpenAssetIO.git
+	branch = main
+[submodule "OpenAssetIO-MediaCreation"]
+	path = OpenAssetIO-MediaCreation
+	url = https://github.com/elliotcmorris/OpenAssetIO-MediaCreation.git
```

Comment:
[Fix] Remember to update (this and commited submodule object)

Reply:
https://github.com/OpenAssetIO/OpenAssetIO-Test-CMake/pull/7/commits/4d07445c49240fd5064a836af5e0f513ff8e19cf